dirty_cat/tests/test_table_vectorizer.py

```diff
@@ -473,7 +473,7 @@ def test_handle_unknown():
     """
     X = _get_clean_dataframe()
     # Test with low cardinality and a StandardScaler for the numeric columns
-    sup_vec = SuperVectorizer(
```

Comment:
I'm sorry, but it's kind of ugly to keep the name "sup_vec".

In a few years people will wonder what it is there for.

Reply:
ooops

Reply:
I replaced it everywhere in dirty_cat (it also appeared in examples)